components/UserForm.js

```diff
@@ -0,0 +1,105 @@
+export default function UserForm({ client, handleSubmit, handleInputChange }) {
+  return (
+    <form onSubmit={handleSubmit}>
+      <label htmlFor="firstName">Name{client.firstName}</label>
```

Comment:
Why is the {client.firstName} in here? At the LastName you don't have that in the label...

Reply:
It was only for testing and i will delete it